test/common/utils/accessible-menus.e2e.spec.js

```diff
@@ -0,0 +1,228 @@
+    client.expect.element('#vetnav-disability').to.not.be.visible;
+    client
+      .focusOn(firstMenuItem)
+      .keys(SPACE)
+      .assert.visible('#vetnav-disability');
+    // TODO: Uncomment this when the code that makes this work is merged in
+    // client.assert.isActiveElement(disFirstItem);
+
+    // Enter should open the sub-menu and focus on the first element if sub-menu exists
+    client.expect.element('#vetnav-healthcare').to.not.be.visible;
+    client
+      .focusOn(secondMenuItem)
+      .keys(ENTER)
+      .assert.visible('#vetnav-healthcare');
+    // TODO: Uncomment this when the code that makes this work is merged in
+    // client.assert.isActiveElement(hcFirstItem);
+
+
+    // -- Non-sub menu buttons -- //
+    const prescriptionsLink = 'a[href="/health-care/prescriptions"]';
+
+    client.click(benefitsButton);
+
+    // Right arrow should do nothing if no sub-menu exists
+    client
+      .focusOn(prescriptionsLink)
+      .keys(RIGHT_ARROW)
+      .assert.visible(prescriptionsLink);
+
+    // TODO: Write these; I'm not sure what to do with them
+    //  Do we just poke them and test the url? Seems like it'd get pretty slow
+    // Space should follow the link if the menu item is a link
+    // Enter should follow the link if the menu item is a link
+
+    client.click(exploreButton); // Open our explore menu back up again
+
+
+    // ---------------------- //
+    // --- Sub-menu tests --- //
+    // ---------------------- //
+
+    client.click(secondMenuItem); // Open up the health care menu
+
+    // Down arrow should move focus to the next item
+    client
+      .focusOn(hcFirstItem)
+      .keys(DOWN_ARROW)
+      .assert.isActiveElement(hcSecondItem);
+    //  with wraparound
+    client
+      .focusOn(hcLastItem)
+      .keys(DOWN_ARROW)
+      .assert.isActiveElement(hcFirstItem);
+
+    // Up arrow should move focus to the previous item
+    client
+      .focusOn(hcSecondItem)
+      .keys(UP_ARROW)
+      .assert.isActiveElement(hcFirstItem);
+    //  with wraparound
+    client
+      .focusOn(hcFirstItem)
+      .keys(UP_ARROW)
+      .assert.isActiveElement(hcLastItem);
+
+    // Right arrow should do nothing
+    client
+      .focusOn(hcFirstItem)
+      .keys(RIGHT_ARROW)
+      .assert.isActiveElement(hcFirstItem);
+
+    // Left arrow should move focus back to the opening menu button
+    client
+      .focusOn(hcFirstItem)
+      .keys(LEFT_ARROW)
+      .assert.isActiveElement(secondMenuItem);
+
+    // TODO: Write these; I'm not sure what to do with them
+    //  Do we just poke them and test the url? Seems like it'd get pretty slow
+    // Space should follow links
+    // Enter should follow links
+
+
+    // --------------------------- //
+    // --- Miscellaneous tests --- //
+    // --------------------------- //
+
+    // Escape should close all menus
+    //  Should focus be doing anything special here? When hitting tab, it goes to the next menubar item
```

Comment:
It looks like focus should return to the toggle element/menubar item for the closed submenu.

Reply:
Ahh, okay. I'll write up another ticket to address that so I don't clutter up this PR. 👍

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7149